lib/stats/processors/student_activity.rb

```diff
@@ -0,0 +1,195 @@
+# encoding: UTF-8
+module Stats
+  module Processors
+    class StudentActivity < ProcessorCore
+
+      sheet_name "Aktivita studentov"
+
+      sheet_split "D3"
+
+      CREDENTIALS = ["AISID", "Login", "Priezvisko", "Meno", ]
+      METRICS_HEADER = ["Návštevy (uč.)",       #"Počet zobrazených otázok",
+                        "Návštevy (test.)",     #"Počet zobrazených správnych odpovedí",
+                        "Evaluator",            #"Počet hodnotení otázok type Evaluation",
+                        "Správne (unikátne)",   #"Počet doteraz správne zodpovedaných otázok",
+                        "Správne",              #"Počet doteraz správne zodpovedaných pokusov",
+                        "Nesprávne",            #"Počet doteraz nesprávne zodpovedaných pokusov",
+                        "Neviem",               #"Počet kliknutí na odpoveď Neviem",
+                        "Pozn."                 #"Počet vložených poznámok"
+                       ]
+
+      def self.week_start(setup, week)
+        setup.first_week_at.to_date + week.number * 7 - 7
```

Comment:
Aha, pozeram, ze taka metoda uz dokonca existuje a robi presne to iste: https://github.com/PewePro/alef-tng/blob/master/app/models/week.rb#L25